friends/src/App.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+
+import FriendsList from './FriendsList';
+import FriendForm from './FriendForm'
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+        friends: [],
+        
+    };
+  }
+  
+  componentDidMount() {
+    axios
+        .get('http://localhost:5000/friends')
+        .then(response => {
+            this.setState(() => ({ friends: response.data }));
+        })
+        .catch(error => {
+          console.error('Ummm...what...!?!', error);
+        });
+  }
+  
+  inputHandler = e => {
+    this.setState({ [e.target.name]: e.target.value })
+  };
+  
+  addFriend = event => {
+    const { name, age, email } = this.state;
+    event.preventDefault();
+    axios.post('http://localhost:5000/friends', {name, age, email})
+    .then(response => {
+      this.setState({ friends: response.data, name: "", age: "", email: "" });
+      window.location.reload();
+    })
+    .catch(err => console.log("Umm...that's a problem", err));
+  };
+  
+  deleteFriend = (id) => {
+    return () => {
+      axios.delete(`http://localhost:5000/friends/${id}`)
+      .then(response => {
+        this.setState({friends: response.data})
+      })
+      .catch(err => console.log("You can't get rid of me that easily!"))
+    }
+
+  }
+  
+  updateHandler = event => {
+    const { name, age, email } = this.state;
+    const id = this.props.friend.id
+    event.preventDefault();
+    this.updateFriendHandler(id, name, age, email)
+  }
+  
+  updateFriendHandler = (id, name, age, email) => {
+    axios.put(`http://localhost:5000/friends/${id}`, {
+      friend: {
```

Comment:
On line 63, you have an object within an object as your second argument. Instead of having friend as a key, just set `{name: name, age: age, email:email}`. Also with ES6 syntax, if the key is equal to the value then you can just write `{name,age,email}`